xray/manager.go

```diff
@@ -108,3 +108,17 @@ func (sm *XrayServicesManager) AddBuildsToIndexing(buildNames []string) error {
 	binMgrService.XrayDetails = sm.config.GetServiceDetails()
 	return binMgrService.AddBuildsToIndexing(buildNames)
 }
+
+// ScanGraph will send Xray the given graph for scan
+func (sm *XrayServicesManager) ScanGraph(params services.XrayGraphScanParams) (string, error) {
```

Comment:
I assume that the returned string is scan ID. Let's add this to the description.